go/engine/account_reset.go

```diff
@@ -153,7 +158,14 @@ func (e *AccountReset) Run(mctx libkb.MetaContext) (err error) {
 		return err
 	}
 	mctx.G().Log.Debug("autoreset/enter result: %s", res.Body.MarshalToDebug())
-	mctx.G().Log.Info("Your account has been added to the reset pipeline.")
+	if alreadyIn {
+		mctx.G().Log.Info("Your account's reset request is now verified.")
```

Comment:
1. can you merge in 714 so that we don't get merge conflicts?
2. we need to call a login UI RPC before returning in this case; lmk if you want help figuring out which one.